composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "ereolen/ereolengo",
+    "description": "eReolen Go",
+    "license": "MIT",
```

Comment:
This is an Drupal project and should this license be `GPL-2.0+` (https://spdx.org/licenses/)